Camera2Extensions/app/src/main/java/com/example/android/camera2/extensions/fragments/CameraFragment.kt

```diff
@@ -651,6 +665,9 @@ class CameraFragment : Fragment(), TextureView.SurfaceTextureListener {
     previewSurface = createPreviewSurface(previewSize)
     stillImageReader = createStillImageReader()
     postviewImageReader = createPostviewImageReader()
+    if (postviewImageReader == null) {
```

Comment:
nit: we can condense this `isPostviewAvailable = postviewImageReader != null`

Reply:
Done